README.md

```diff
@@ -79,30 +79,44 @@ In order to avoid installing ROS on your local machine, we will be using a docke
 ### Installation
 
 1. Clone the repo
```

Comment:
The numered list is not being display correctly in github but is not a big issue